src/plugins/mer/mervirtualboxmanager.h

```diff
@@ -52,6 +54,11 @@ class VirtualMachineInfo
     QList<quint16> qmlLivePorts;
     QStringList macs;
     bool headless;
+    int memorySizeMb;
+    int cpuCount;
+    QString vdiPath;
+    int vdiCapacity;
+    int vdiSizeOnDisk;
```

Comment:
Unless I missed something we only query vdiSizeOnDisk once, then store it in QtC settings and never update. That's not right because it changes continuously as the VM is used. I would prefer removing that instead of fixing - it is not important to our use case anyway. The only moment the user would care about size-on-disk is when the disk gets full, but then there is more to check than our two VMs and more suitable tools are likely to be used. Note that this value generally does not correspond to the remaining disk space inside the VM - if this was your motivation for adding this we would need to ssh into the device and use `du` instead.